### PR TITLE
Like Feature Improvement

### PR DESCRIPTION
**Pull request by DavidLiang01**
Improvements: Only allow user to like once. When user liked a post, the heart icon is red #69

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -209,13 +209,23 @@ async function initialize_server(con) {
 
       router.post('/fetch_feed', function(req,res,next) {
         if (req.session && req.session.email) {
-          con.query("SELECT * FROM posts LEFT JOIN users ON posts.userid=users.uid ORDER BY posts.id DESC LIMIT 10", [], function (err, result) {
+          // con.query("SELECT * FROM posts LEFT JOIN users ON posts.userid=users.uid ORDER BY posts.id DESC LIMIT 10", [], function (err, result) {
```
Comment (jeffrey-w) [inline main.js:212]: Move/remove commented out code (Code Style Guide 8)
```suggestion
```

File: client/dashboard.html [hunk 2]
```diff
@@ -179,6 +181,26 @@
...
    }
 
+   function likario(postid){
+    var xhttp = new XMLHttpRequest();
+    xhttp.onreadystatechange = function() {
+       if (this.readyState == 4 && this.status == 200) {
+         var result = this.response;
+         console.log(result);
+         if(result == "true")
+         {
```
Comment (jeffrey-w) [inline client/dashboard.html:191]: Open statement blocks on the same line as their declaration (Code Style Guide 4)

File: main.js (hunk 1 above)
Comment (DavidLiang01) [reply main.js:212]: I commented this line because I want to keep it if we face a bug later , so that I remember what was the original query

Comment (jeffrey-w) [reply main.js:212]: Old code may be located in the repo history (this is one of the reasons version control exists).

**Review by jeffrey-w — APPROVED** [review]
A lot of violations of our style guide make this harder to read than it should be (mostly very long lines of text for your SQL). Nevertheless, it looks functionally sound, and so I approve.

**Comment by DavidLiang01** [comment]
This SQL query, cant do anything if you want it to work

File: main.js (hunk 1 above)
Comment (DavidLiang01) [reply main.js:212]: What do you mean by repo history

Comment (jeffrey-w) [reply main.js:212]: Git stores every commit we've ever made. If you want to revert to this commented out code it may be found in one of these commits.

**Pull request merged**